integrations/google.py

```diff
@@ -0,0 +1,114 @@
+import random
+import string
+from datetime import timedelta
+
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+from google_auth_oauthlib.flow import Flow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from google.oauth2.credentials import Credentials
+
+from django.conf import settings
+
+
+class GoogleCalendar:
+    def __init__(self, credentials, calendar_id='primary'):
+        self.service = build('calendar', 'v3', credentials=credentials)
+        self.calendar_id = calendar_id
+
+    @property
+    def events(self):
```

Comment:
I'm curious, why not just create `self.events = self.service.events()` in the init? 🤔 I wonder what calling `events()` each time you want `self.events` does.

Reply:
Good point, I should do that instead.